circus-train-s3-mapreduce-cp/src/main/java/com/hotels/bdp/circustrain/s3mapreducecp/ConfigurationVariable.java

```diff
@@ -32,6 +34,7 @@ private Constants() {}
 
 public enum ConfigurationVariable {
 
+  ASSUME_ROLE(AssumeRoleCredentialProvider.ASSUME_ROLE_PROPERTY_NAME, null),
```

Comment:
I'm using the credential provide property here cause that's what we really need to be setting and we got a dependency on ct-aws not the other way around.